src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -173,6 +183,18 @@ export const TradeConfirm = ({ history }: RouterProps) => {
                 </HelperTooltip>
                 <Row.Value>{toFiat(0)}</Row.Value>
               </Row>
+              {gasFeeToTradeRatioPercentage > gasFeeToTradeRatioPercentageThreshold ? (
```

Comment:
nit pick - i generally prefer if logic like this is encapsulated in it's own boolean, e.g.

```
const isFeeRatioOverThreshold = gasFeeToTradeRatioPercentage > gasFeeToTradeRatioPercentageThreshold
isFeeRatioOverThreshold && (<Component bits...
```

but this is fine

Reply:
I like that too, I've switched it over